src/pages/library/Code.js

```diff
@@ -51,89 +50,97 @@ export const makeWorkflowCard = ({ method, onClick }) => {
   ])
 }
 
-const logoTile = ({ logoFile, style = {} }) => div({
-  style: {
-    flexShrink: 0,
-    backgroundImage: `url(${logoFile})`,
-    backgroundRepeat: 'no-repeat', backgroundPosition: 'center', backgroundColor: 'white',
-    backgroundSize: 27,
-    width: 37, height: 37,
-    marginRight: 13,
-    ...style
-  }
-})
+const LogoTile = ({ logoFile, ...props }) => {
+  return div(_.merge({
+    style: {
+      flexShrink: 0,
+      backgroundImage: `url(${logoFile})`,
+      backgroundRepeat: 'no-repeat', backgroundPosition: 'center',
+      backgroundSize: 27,
+      width: 37, height: 37,
+      marginRight: 13
+    }
+  }, props))
+}
 
-export const dockstoreTile = () => div({ style: { display: 'flex' } }, [
-  logoTile({ logoFile: dockstoreLogo }),
-  div([
-    h(Link, { href: `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=wdl&searchMode=files` }, 'Dockstore'),
-    div(['Browse WDL workflows in Dockstore, an open platform used by the GA4GH for sharing Docker-based workflows'])
+export const DockstoreTile = () => {
```

Comment:
Now that config is available at load, is there any reason for these not to just be straight declarations?

Reply:
It avoids having this code work differently from other components just because it's not dynamic (right now). There's less cognitive overhead, and on the off chance we need to add a prop later, the refactoring is that much more straightforward.